o-fish-ios/Views/Components/Picker/SearchBarView.swift

```diff
@@ -30,11 +31,10 @@ struct SearchBarView: View {
             HStack(alignment: .center) {
                 GlassIconView()
 
-                TextField(LocalizedStringKey(placeholder), text: $searchText, onEditingChanged: { _ in
-                    print("onEdit")
-                }, onCommit: {
-                    self.onCommit?()
-                })
+                TextField(LocalizedStringKey(placeholder),
+                    text: $searchText,
+                    onEditingChanged: onEditingChanged,
+                    onCommit: self.onCommit)
```

Comment:
without "self"

Reply:
inside the block it is need to use self

Reply:
But it's not a block. We have onEditingChanged closure without "self" and onCommit closure with "self"